### PR TITLE
Feature/ship tables

### LINKED ISSUE #31: bundle latest supported playfield with vpweb
bundle the latest supported playfield with vpweb. so we're sure we test all with the same playfield and avoid confusion.

### PR DESCRIPTION
**Pull request by freezy**
This adds two tables to the web app so we can quickly test and be sure we test the same tables.

There were also some UI updates.

image

This PR closes #31.

### CODE REVIEW

**Comment by neophob** [comment]
Sir, this kicks ass!

Tested the two tables, some small inputs:
- a label for each table would make sense as it would explain what we see here
- on the second table i see the error `THREE.BufferGeometry.computeBoundingSphere(): Computed radius is NaN. The "position" attribute is likely to have NaN values. `

File: src/index.sass [hunk 1]
```diff
@@ -45,8 +46,24 @@ body,html
 	height: 100%
 body
 	margin: 0
-	background-color: #333
 	position: relative
+	//background-color: #333
```
Comment (neophob) [inline src/index.sass]: minor: remove commented code

**Comment by neophob** [comment]
the initialisation of the mm table seems to miss the switch setting - thus we see this error in the dmd

File: src/index.sass (hunk 1 above)
Comment (freezy) [reply src/index.sass]: I let this in because it's the background used to make the screenshots.

**Comment by freezy** [comment]
- I thought about labels too. Might add them in the future, along with an explanation perhaps.
- What was the switch settings thing again?

Thanks for the review, will merge and address your points in a separate PR.

**Pull request merged**